documentation/modules/upgrading/con-upgrade-custom-resources.adoc

```diff
@@ -0,0 +1,104 @@
+// This module is included in the following assemblies:
+//
+// assembly-upgrade-resources.adoc
+
+[id='con-upgrade-custom-resources-{context}']
+
+= API versioning
+
+[role="_abstract"]
+Custom resources are created as instances of APIs added by CRDs to extend Kubernetes resources.
```

Comment:
This is just a suggestion...

```suggestion
Custom resources are manipulated using APIs added to Kubernetes by CRDs. 
Put another way, CRDs extend the Kubernetes API to allow creation of custom resources.
```

Reply:
Edited and controlled sounds good: _Custom resources are edited and controlled using APIs added to Kubernetes by CRDs._

Reply:
> Custom resources are edited and controlled using APIs added to Kubernetes by CRDs.
> Put another way, CRDs extend the Kubernetes API to allow creation of custom resources.